<?xml version="1.0" encoding="UTF-8"?>
<!-- ===============================================
  Copyright (C) 2014 ADVENTURE Project
  All Rights Reserved
================================================ -->
<ui version="4.0">
    <class>TestDataDialog</class>
    <widget class="QDialog" name="TestDataDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>500</width>
                <height>265</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Test data</string>
        </property>
        <widget class="QLabel" name="label_3">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>20</y>
                    <width>250</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>Intersections:</string>
            </property>
        </widget>
        <widget class="QSpinBox" name="spinBoxMaxX">
            <property name="geometry">
                <rect>
                    <x>300</x>
                    <y>20</y>
                    <width>71</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
                <number>0</number>
            </property>
            <property name="maximum">
                <number>999999</number>
            </property>
        </widget>
        <widget class="QSpinBox" name="spinBoxMaxY">
            <property name="geometry">
                <rect>
                    <x>400</x>
                    <y>20</y>
                    <width>71</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
                <number>0</number>
            </property>
            <property name="maximum">
                <number>999999</number>
            </property>
        </widget>
        <widget class="QLabel" name="label_4">
            <property name="geometry">
                <rect>
                    <x>380</x>
                    <y>20</y>
                    <width>21</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>x</string>
            </property>
        </widget>
        <widget class="QSpinBox" name="spinBoxCycle4Y">
            <property name="geometry">
                <rect>
                    <x>400</x>
                    <y>80</y>
                    <width>71</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
                <number>0</number>
            </property>
            <property name="maximum">
                <number>999999</number>
            </property>
        </widget>
        <widget class="QSpinBox" name="spinBoxCycle4X">
            <property name="geometry">
                <rect>
                    <x>300</x>
                    <y>80</y>
                    <width>71</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
                <number>0</number>
            </property>
            <property name="maximum">
                <number>999999</number>
            </property>
        </widget>
        <widget class="QLabel" name="label_5">
            <property name="geometry">
                <rect>
                    <x>380</x>
                    <y>80</y>
                    <width>21</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>x</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_6">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>80</y>
                    <width>250</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>Section interval (2 lanes each way):</string>
            </property>
        </widget>
        <widget class="QSpinBox" name="spinBoxCycle8Y">
            <property name="geometry">
                <rect>
                    <x>400</x>
                    <y>110</y>
                    <width>71</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
                <number>0</number>
            </property>
            <property name="maximum">
                <number>999999</number>
            </property>
        </widget>
        <widget class="QSpinBox" name="spinBoxCycle8X">
            <property name="geometry">
                <rect>
                    <x>300</x>
                    <y>110</y>
                    <width>71</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
                <number>0</number>
            </property>
            <property name="maximum">
                <number>999999</number>
            </property>
        </widget>
        <widget class="QLabel" name="label_7">
            <property name="geometry">
                <rect>
                    <x>380</x>
                    <y>110</y>
                    <width>21</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>x</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_8">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>110</y>
                    <width>250</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>Section interval (4 lanes each way):</string>
            </property>
        </widget>
        <widget class="QPushButton" name="buttonCancel">
            <property name="geometry">
                <rect>
                    <x>250</x>
                    <y>210</y>
                    <width>101</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="text">
                <string>Cancel</string>
            </property>
            <property name="default">
                <bool>false</bool>
            </property>
        </widget>
        <widget class="QPushButton" name="buttonOk">
            <property name="geometry">
                <rect>
                    <x>370</x>
                    <y>210</y>
                    <width>101</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="text">
                <string>OK</string>
            </property>
            <property name="default">
                <bool>true</bool>
            </property>
        </widget>
        <widget class="QLabel" name="label_9">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>50</y>
                    <width>250</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>Section length(m):</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_10">
            <property name="geometry">
                <rect>
                    <x>380</x>
                    <y>50</y>
                    <width>21</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>x</string>
            </property>
        </widget>
        <widget class="QDoubleSpinBox" name="dbSpinDistX">
            <property name="geometry">
                <rect>
                    <x>300</x>
                    <y>50</y>
                    <width>71</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
                <number>1</number>
            </property>
            <property name="minimum">
                <double>-1000000000.000000000000000</double>
            </property>
            <property name="maximum">
                <double>1000000000.000000000000000</double>
            </property>
            <property name="singleStep">
                <double>10.000000000000000</double>
            </property>
        </widget>
        <widget class="QDoubleSpinBox" name="dbSpinDistY">
            <property name="geometry">
                <rect>
                    <x>400</x>
                    <y>50</y>
                    <width>71</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
                <number>1</number>
            </property>
            <property name="minimum">
                <double>-1000000000.000000000000000</double>
            </property>
            <property name="maximum">
                <double>1000000000.000000000000000</double>
            </property>
            <property name="singleStep">
                <double>10.000000000000000</double>
            </property>
        </widget>
        <widget class="QLabel" name="label_11">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>140</y>
                    <width>250</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>Section interval (branch):</string>
            </property>
        </widget>
        <widget class="QSpinBox" name="spinBoxCycleBrY">
            <property name="geometry">
                <rect>
                    <x>400</x>
                    <y>140</y>
                    <width>71</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
                <number>0</number>
            </property>
            <property name="maximum">
                <number>999999</number>
            </property>
        </widget>
        <widget class="QSpinBox" name="spinBoxCycleBrX">
            <property name="geometry">
                <rect>
                    <x>300</x>
                    <y>140</y>
                    <width>71</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
                <number>0</number>
            </property>
            <property name="maximum">
                <number>999999</number>
            </property>
        </widget>
        <widget class="QLabel" name="label_12">
            <property name="geometry">
                <rect>
                    <x>380</x>
                    <y>140</y>
                    <width>21</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>x</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_13">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>170</y>
                    <width>250</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>Fluctuation(m):</string>
            </property>
        </widget>
        <widget class="QSpinBox" name="spinBoxRnd">
            <property name="geometry">
                <rect>
                    <x>300</x>
                    <y>170</y>
                    <width>71</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Sans Serif</family>
                    <pointsize>8</pointsize>
                </font>
            </property>
            <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
            </property>
            <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
                <number>0</number>
            </property>
            <property name="maximum">
                <number>999999</number>
            </property>
        </widget>
    </widget>
    <tabstops>
        <tabstop>buttonCancel</tabstop>
        <tabstop>spinBoxMaxX</tabstop>
        <tabstop>spinBoxMaxY</tabstop>
        <tabstop>dbSpinDistX</tabstop>
        <tabstop>dbSpinDistY</tabstop>
        <tabstop>spinBoxCycle4X</tabstop>
        <tabstop>spinBoxCycle4Y</tabstop>
        <tabstop>spinBoxCycle8X</tabstop>
        <tabstop>spinBoxCycle8Y</tabstop>
        <tabstop>spinBoxCycleBrX</tabstop>
        <tabstop>spinBoxCycleBrY</tabstop>
        <tabstop>spinBoxRnd</tabstop>
        <tabstop>buttonOk</tabstop>
    </tabstops>
    <resources/>
    <connections/>
</ui>
